validator_client/src/lib.rs

```diff
@@ -183,7 +184,16 @@ impl<T: EthSpec> ProductionValidatorClient<T> {
             log.clone(),
         )
         .await
-        .map_err(|e| format!("Unable to initialize validators: {:?}", e))?;
+        .map_err(|e| {
+            match e {
+                UnableToOpenVotingKeystore(err) => {
+                    format!("Unable to initialize validators: {:?}, If you have recently moved the location of your data directory \
```

Comment:
```suggestion
                    format!("Unable to initialize validators: {:?}. If you have recently moved the location of your data directory \
```

We've been trying to remove `,` from the logs since it's challenging for indexing log systems (e.g., ELK) that parse our `key1: value1, key2: value:2` format.